#! /bin/sh /usr/share/dpatch/dpatch-run
## 05-disable-buildcmodule.dpatch by Chris Lamb <chris@chris-lamb.co.uk>
##
## DP: Ensure that builcmodule does not remove the library inside /usr/lib.
## DP: See #471844. Patch by Mark Wooding <mdw@distorted.org.uk>.

@DPATCH@
diff -urNad py-0.9.1~/py/misc/buildcmodule.py py-0.9.1/py/misc/buildcmodule.py
--- py-0.9.1~/py/misc/buildcmodule.py	2008-03-28 10:40:38.000000000 +0000
+++ py-0.9.1/py/misc/buildcmodule.py	2008-04-04 00:09:12.000000000 +0100
@@ -33,10 +33,7 @@
 
     # XXX argl! we need better "build"-locations alltogether!
     if lib.check():
-        try:
-            lib.remove()
-        except EnvironmentError:
-            pass # XXX we just use the existing version, bah
+        pass # XXX we just use the existing version, bah
 
     if not lib.check():
         c = py.io.StdCaptureFD()
